integration-tests/emulated/chains/relays/kusama/Cargo.toml

```diff
@@ -25,4 +25,4 @@ emulated-integration-tests-common = { workspace = true }
 
 # Runtimes
 kusama-runtime-constants = { workspace = true, default-features = true }
-kusama-runtime = { workspace = true }
+kusama-runtime = { workspace = true, default-features = true }
```

Comment:
```suggestion
kusama-runtime = { workspace = true }
```
We never disabled them in the workspace, zepter sad.

Reply:
ah, for kusama not, for polkadot actually it is disabled

Reply:
Actually that is what has been causing issues 🙃 removed if from Polkadot runtime now.